ALGORITHMS.md

```diff
@@ -11,6 +11,9 @@ selected channel. This is only for testing and re-initialization.
 
 Parameters:
 * `mode`: "random"
+* `args`:
+  * `setDifferentChannelPerAp`: If true, will set a different random channel per AP. If false, it will set the same random channel for all APs.
+    * values: true | false (default: false)
```

Comment:
- "args" shouldn't be a separate section, it should all go under "Parameters" ("mode" is passed the same way)
- You need to indent list items with 4 spaces (not 2) for better Markdown renderer support (from what I understand)
- see suggested styling for values (commas, code font)

```suggestion
* `setDifferentChannelPerAp`: If true, will set a different random channel per AP. If false, it will set the same random channel for all APs.
    * Values: `true`, `false` (default: `false`)
```